translations/all-messages.json

```diff
@@ -46,12 +46,6 @@
   "listing-card-prices.ethereumCurrencyAbbrev": "ETH",
   "listing-card-prices.sold": "Sold",
   "listing-card.featured": "Featured",
-  "listing-create.forSaleLabel": "For Sale",
```

Comment:
Why did those strings get deleted ? Don't we still need them for the listing creation page ?

Reply:
Good question will look into that. It could be that I have removed the hardcoded translations from `listing-create` into a separate file, because `searchbar` also requires that schema/translations. Just so that we do not have the data duplicated...